code/modules/antagonists/heretic/items/heretic_necks.dm

```diff
@@ -9,6 +9,100 @@
 	. = ..()
 	AddElement(/datum/element/heretic_focus)
 
+/obj/item/clothing/neck/heretic_focus/crimson_focus
+	name = "Crimson Focus"
+	desc = "A blood-red focusing glass that provides a link to the world beyond, and worse. Its eye is constantly twitching and gazing in all directions. It almost seems to be silently screaming..."
+	icon_state = "crimson_focus"
+	/// The aura healing component. Used to delete it when taken off.
+	var/datum/component/component
+
+
+/obj/item/clothing/neck/heretic_focus/crimson_focus/equipped(mob/living/user, slot)
+	. = ..()
+	if(!(slot & ITEM_SLOT_NECK))
+		return
+
+	var/team_color = COLOR_ADMIN_PINK
+	if(IS_CULTIST(user))
+		var/datum/action/innate/cult/blood_magic/magic_holder = locate() in user.actions
+		team_color = COLOR_CULT_RED
+		magic_holder.magic_enhanced = TRUE
+	if(IS_HERETIC_OR_MONSTER(user))
+		for(var/datum/action/cooldown/spell/spell_action in user.actions)
+			spell_action.cooldown_time *= 0.5
+		team_color = COLOR_GREEN
+	else
+		team_color = pick(COLOR_CULT_RED, COLOR_GREEN)
+
+	user.add_traits(list(TRAIT_MANSUS_TOUCHED, TRAIT_BLOODY_MESS), REF(src))
+	to_chat(user, span_alert("Your heart takes on a strange yet soothing irregular rhythm, and your blood feels significantly less viscous than it used to be. You're not sure if that's a good thing."))
+	component = user.AddComponent( \
+		/datum/component/aura_healing, \
+		range = 3, \
+		brute_heal = 1, \
+		burn_heal = 1, \
+		blood_heal = 2, \
+		suffocation_heal = 5, \
+		simple_heal = 0.6, \
+		requires_visibility = FALSE, \
+		limit_to_trait = TRAIT_MANSUS_TOUCHED, \
+		healing_color = team_color, \
+	)
+
+/obj/item/clothing/neck/heretic_focus/crimson_focus/dropped(mob/living/user)
+	. = ..()
+
+	if(!istype(user))
+		return
+
+	if(HAS_TRAIT_FROM(user, TRAIT_MANSUS_TOUCHED, REF(src)))
+		to_chat(user, span_notice("Your heart and blood return to their regular old rhythm and flow."))
+
+	if(IS_HERETIC_OR_MONSTER(user))
+		for(var/datum/action/cooldown/spell/spell_action in user.actions)
+			if(spell_action.cooldown_time == initial(spell_action.cooldown_time))
+				return
+			spell_action.cooldown_time *= 2
```

Comment:
This is still imperfect, there's a few locations (namely after ascension) that we half cooldown of some of a heretic's spells. 

Really it's not so bad keeping a var on the item `var/is_active = FALSE`, setting it to `TRUE` when we apply the buffs and checking `if(!is_active) return` in `dropped`